src/pybind/mgr/dashboard/controllers/multi_cluster.py

```diff
@@ -0,0 +1,72 @@
+from . import APIDoc, APIRouter, CreatePermission, UpdatePermission, ReadPermission, Endpoint, EndpointDoc, RESTController
+import requests
+from ..exceptions import DashboardException
+
+from ..settings import Settings
+from ..security import Scope
+
+import logging
+logger = logging.getLogger('routes')
+
+
+@APIRouter('/multicluster', Scope.CONFIG_OPT)
+@APIDoc('Multi Cluster Route Management API', 'Multi Cluster Route')
+class MultiClusterRoute(RESTController):
+    def _proxy(self, method, path, api_name, params=None, payload=None, verify=False):
+
+        config2 = Settings.REMOTE_CLUSTER_URLS
+        matching_dict = None
+        base_url = ''
+        api_token = ''
+        config = Settings.MULTICLUSTER_CONFIG
+        if config2 is not None and config is not None:
+            matching_dict = next((item for item in config2 if item.get('remoteClusterUrl') == config['current_url']), None)
```

Comment:
even for the POC purpose, we shouldn't need 2 settings. We just need one setting which will hold all the config details of all the clusters that are joined, which will simplify things